nuimo.py

```diff
@@ -341,7 +350,11 @@ def stop_discovery(self):
 
     @staticmethod
     def filter_nuimos(devices):
-        return [addr for addr, attrs in devices.items() if attrs.get('name') == 'Nuimo']
+        return [
+            addr for addr, attrs in
+            devices.items() if attrs and
+            attrs.get('name') == 'Nuimo'
+        ]
```

Comment:
I can see that you added a null-check for `attrs`. The change however doesn't look properly formatted to me, should be either indented, but I prefer the previous one-liner formatting.